site/content/includes/quickstart-prometheus-steps.md

```diff
@@ -0,0 +1,63 @@
+This quickstart uses [the textfile collector feature](https://github.com/prometheus/node_exporter#textfile-collector) of the Prometheus node exporter to export metrics to Prometheus that M3 then ingests. To follow the next steps, [download node_exporter](https://github.com/prometheus/node_exporter#installation-and-usage)
+
+#### Configure and Start Prometheus
```

Comment:
I wonder if its worth starting node exporter first, otherwise youll see a bunch of errors in prom.

Reply:
@benraskin92 I tested the steps and their order quite a lot, and this was the order that resulted in the least errors, and worked for me. Did you try the steps yourself?